osbs/conf.py

```diff
@@ -510,3 +511,18 @@ def get_low_priority_node_selector(self):
             nodeselector = dict([k.strip(), v.strip()] for (k, v) in raw_nodeselector.items())
 
         return nodeselector
+
+    def get_equal_labels(self):
+        equal_labels = []
+        equal_labels_str = self._get_value("equal_labels", self.conf_section,
+                                           "equal_labels")
+        if equal_labels_str:
+            if re.match('^[^:, ]+(:[^:, ]+)+\s*(,\s*[^:, ]+(:[^:, ]+\s*)+)*$', equal_labels_str):
```

Comment:
ok I'll add comment, it is just nice to validate it via regs, but for parsing is better just string func